pom.xml

```diff
@@ -27,8 +27,35 @@
 
     <groupId>com.uber.tchannel</groupId>
     <artifactId>tchannel</artifactId>
+    <version>0.1.1-SNAPSHOT</version>
```

Comment:
kind of. we 'tagged a release' but since we weren't uploading / releasing anything to the Central Repository it was kind of a fake release.